doc/content/reference/configuration/_index.md

```diff
@@ -8,6 +8,10 @@ aliases: [/guides/getting-started/configuration]
 
 <!--more-->
 
+{{< note >}}
+These configuration methods only apply to {{% tts %}} Open Source and Enterprise deployments installed by following the [Installation]({{< ref "getting-started/installation" >}}) guide. AWS Launcher deployments can be configured using the [CloudFormation template settings]({{< ref "/getting-started/aws/ami/deployment-guide#deployment-using-aws-cloud-formation" >}}). Managed deployments such as Cloud and Community Edition are configured for you by [The Things Industries](mailto:info@thethingsindustries.com).
```

Comment:
Remove contact information here, we're not going to reply to emails regarding stack configuration through info@

For Cloud, that would go through support. For Community, that would go through the forums.
```suggestion
These configuration methods only apply to {{% tts %}} Open Source and Enterprise deployments installed by following the [Installation]({{< ref "getting-started/installation" >}}) guide. AWS Launcher deployments can be configured using the [CloudFormation template settings]({{< ref "/getting-started/aws/ami/deployment-guide#deployment-using-aws-cloud-formation" >}}). Managed deployments such as Cloud and Community Edition are configured for you by The Things Industries.
```